services/net/scheduler/SchedulerManager.cs

```diff
@@ -83,19 +83,19 @@ public override async Task RunAsync()
                         {
                             this.Logger.LogInformation("Scheduled event '{id}:{name}' is running", eventTypeId, scheduledEvent.Name);
 
-                            if (ev.EventType == EventScheduleType.Report)
-                                await GenerateReportRequestAsync(scheduledEvent);
-                            else if (ev.EventType == EventScheduleType.Notification)
-                                await GenerateNotificationRequestAsync(scheduledEvent);
-                            else
-                                await GenerateEventScheduleRequestAsync(scheduledEvent);
-
                             await this.Api.HandleConcurrencyAsync<EventScheduleModel?>(async () =>
                             {
                                 scheduledEvent = await this.Api.GetEventScheduleAsync(scheduledEvent.Id) ?? throw new NoContentException($"Event schedule {scheduledEvent.Id}:{scheduledEvent.Name} does not exist.");
                                 scheduledEvent.RequestSentOn = DateTime.UtcNow;
                                 return await this.Api.UpdateEventScheduleAsync(scheduledEvent, false);
                             });
+
+                            if (ev.EventType == EventScheduleType.Report)
```

Comment:
Change the order.  Now it will update the RequestSentOn before asking for the report to run.  The hope is this will stop multiple runs.